content/en/docs/reference/instrumentation/slis.md

```diff
@@ -0,0 +1,91 @@
+         StabilityLevel: k8smetrics.ALPHA,
+      },
+      []string{"name", "type"},
+   )
+
+   // healthchecksTotal is a Prometheus Counter metrics used for counting the results of a k8s healthcheck.
+   healthchecksTotal = k8smetrics.NewCounterVec(
+      &k8smetrics.CounterOpts{
+         Namespace:      "kubernetes",
+         Name:           "healthchecks_total",
+         Help:           "This metric records the results of all healthcheck.",
+         StabilityLevel: k8smetrics.ALPHA,
+      },
+      []string{"name", "type", "status"},
+   )
+)
+```
+
+The gauge data looks like this:
+
+```shell
+# HELP kubernetes_healthcheck [ALPHA] This metric records the result of a single healthcheck.
+# TYPE kubernetes_healthcheck gauge
+kubernetes_healthcheck{name="autoregister-completion",type="healthz"} 1
+kubernetes_healthcheck{name="autoregister-completion",type="readyz"} 1
+kubernetes_healthcheck{name="etcd",type="healthz"} 1
+kubernetes_healthcheck{name="etcd",type="readyz"} 1
+kubernetes_healthcheck{name="etcd-readiness",type="readyz"} 1
+kubernetes_healthcheck{name="informer-sync",type="readyz"} 1
+kubernetes_healthcheck{name="log",type="healthz"} 1
+kubernetes_healthcheck{name="log",type="readyz"} 1
+kubernetes_healthcheck{name="ping",type="healthz"} 1
+kubernetes_healthcheck{name="ping",type="readyz"} 1
+```
+
+While the counter data looks like this:
+
+```shell
+# HELP kubernetes_healthchecks_total [ALPHA] This metric records the results of all healthcheck.
+# TYPE kubernetes_healthchecks_total counter
+kubernetes_healthchecks_total{name="autoregister-completion",status="error",type="readyz"} 1
+kubernetes_healthchecks_total{name="autoregister-completion",status="success",type="healthz"} 15
+kubernetes_healthchecks_total{name="autoregister-completion",status="success",type="readyz"} 14
+kubernetes_healthchecks_total{name="etcd",status="success",type="healthz"} 15
+kubernetes_healthchecks_total{name="etcd",status="success",type="readyz"} 15
+kubernetes_healthchecks_total{name="etcd-readiness",status="success",type="readyz"} 15
+kubernetes_healthchecks_total{name="informer-sync",status="error",type="readyz"} 1
+kubernetes_healthchecks_total{name="informer-sync",status="success",type="readyz"} 14
+kubernetes_healthchecks_total{name="log",status="success",type="healthz"} 15
+kubernetes_healthchecks_total{name="log",status="success",type="readyz"} 15
+kubernetes_healthchecks_total{name="ping",status="success",type="healthz"} 15
+kubernetes_healthchecks_total{name="ping",status="success",type="readyz"} 15
+```
+
+## Using this data
+
+The component SLIs metrics endpoint is intended to be scraped at a high frequency. Scraping
+at a high frequency means that you end up with greater granularity of the gauge's signal, which
+can be then used to calculate SLOs. The `metrics/slis` endpoint provides the raw data necessary
+to calculate an availability SLO for the respective Kubernetes component. 
```

Comment:
(nit)
What's a high frequency? Are you thinking tens of scrapes per minute, or tens of scrapes per second?